### PR TITLE
[rllib] Add type annotations for evaluation/, env/ packages

### PR DESCRIPTION
**Pull request by ericl**
The remaining packages can be addressed in future PRs: agent/, policy/, models/, offline/, utils/

### CODE REVIEW

**Comment by richardliaw** [comment]
amazing; do you think we should lint this?

**Comment by ericl** [comment]
I think the main value add here is documentation and not correctness, but
we can certainly consider adding lint in the future.

File: rllib/env/__init__.py [hunk 1]
```diff
@@ -1,5 +1,3 @@
-from typing import Any
-
 from ray.rllib.env.base_env import BaseEnv
```
Comment (sven1977) [inline rllib/env/__init__.py:1]: We should add Unity3DEnv here as well:
from ray.rllib.env.unity3d_env import Unity3DEnv

File: rllib/env/__init__.py [hunk 2]
```diff
@@ -10,9 +8,6 @@
 from ray.rllib.env.policy_client import PolicyClient
 from ray.rllib.env.policy_server_input import PolicyServerInput
 
-# Represents one of the env types in this package.
-EnvType = Any
-
 __all__ = [
     "BaseEnv",
```
Comment (sven1977) [inline rllib/env/__init__.py:12]: Add "Unity3DEnv",

File: rllib/utils/types.py [hunk 3]
```diff
@@ -0,0 +1,79 @@
+from typing import Any, Dict, Union, Tuple
```
Comment (sven1977) [inline rllib/utils/types.py:1]: Nice!

**Review by sven1977 — APPROVED** [review]
Looks great. Could you just add the `Unit3DEnv` to its init file?
Thanks

**Pull request merged**